dev-tools/.gitattributes

```diff
@@ -0,0 +1,10 @@
+.* export-ignore
```

Comment:
Why not ignoring the whole directory in the root `.gitattributes` file?

Reply:
Because we need `dev-tools/ci-integration.sh` in distribution and we have test for it, the one [failing](https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/611568199) on first commit.

Also we cannot make this:
```
/dev-tools/
!/dev-tools/ci-integration.sh
```
because it's not allowed:
```
warning: Negative patterns are ignored in git attributes
Use '\!' for literal leading exclamation.
```

Reply:
This looks a bit overkill to me. What about moving the file somewhere else?

Reply:
To project's root?

Reply:
Yes I guess. It seems required by the `--help` CLI option, it's not really a dev tool.